test/parallel/test-child-process-fork-exec-path.js

```diff
@@ -59,7 +59,9 @@ if (process.env.FORK) {
     assert.deepStrictEqual(msg, recv);
   }));
   child.on('exit', common.mustCall(function(code) {
-    fs.unlinkSync(copyPath);
-    assert.strictEqual(code, 0);
+    // Give the process just a tick more time to release the file handle.
+    fs.unlink(copyPath, common.mustCall(() => {
```

Comment:
Maybe just move the `unlinkSync()` call to a `process.on('exit', ...)` handler? The way this moves the check for the value of `code` into an unrelated callback seems to mix concerns in a way we might be best to avoid?

Reply:
Done. Also replaced `unlink()` with the more "graceful" `tmpdir.refresh()`